### ISSUE TITLE
Docs for find_peaks_cwt are incorrect

### ISSUE THREAD

**Issue by Arnoques** [scipy/scipy#6922]
Docs for scipy.signal.find_peaks_cwt state that the default for `gap_thresh` is 2, but the code says (https://github.com/scipy/scipy/blob/master/scipy/signal/_peak_finding.py#L509)

```
if gap_thresh is None:
    gap_thresh = np.ceil(widths[0]) 
```
Thanks

**Comment by souravsingh** [comment]
@rgommers I wish to work on the issue. We have to update the docstring for `gap_thresh`, correct?

**Comment by ashwinpathak20** [comment]
Since, this issue remains to be unsolved , I wish to claim this issue.

**Comment by ashwinpathak20** [comment]
Commit d1ec9438284992fbac80299ced9c1e010bdad091 closes #6922

**Comment by ashwinpathak20** [comment]
#6922 Commit  364aa77a52eac66a3c46485801334db9e9aef07a
